src/Symfony/Component/Form/ChoiceList/Loader/CallbackChoiceLoader.php

```diff
@@ -40,7 +40,7 @@ public function __construct(callable $callback)
     /**
      * {@inheritdoc}
      */
-    public function loadChoiceList($value = null)
+    public function loadChoiceList(?callable $value = null)
```

Comment:
I think all nullable types can be removed if they already have `null` as default

Reply:
Yes, that would be consistent with the rest of the codebase at least.
```suggestion
    public function loadChoiceList(callable $value = null)
```

Reply:
@yceruto , this is debatable. I cannot find the proof but I heard that relying on default nulls without adding `?` will be deprecated since it's an inconsistency.

Reply:
If we are talking about the current Symfony codebase consistency, then I agree

Reply:
Yes, we should remove `?` in this case for now. See https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3883#issue-338880999

Reply:
> I heard that relying on default nulls without adding `?` will be deprecated since it's an inconsistency.

I doubt it because that would create strange edge-cases for applications that aim at broad compatibility ranges. But if that should ever happen, this can be fixed by a tool. Until then, we should aim for consistency.